.github/workflows/main.yaml

```diff
@@ -25,7 +25,7 @@ jobs:
   publish:
     needs: package
     outputs:
-      publishedVersion: ${{ steps.check-release.outputs.version }}
+      published_version: ${{ steps.get-version.outputs.info }}
```

Comment:
the problem was that check-release only returns a version when it doesn't already exist - makes it impossible to retry the pipeline for image building